src/Compilers/CSharp/Portable/BoundTree/BoundExpression.cs

```diff
@@ -255,11 +233,11 @@ public BoundIndexerAccess Update(bool useSetterForDefaultArgumentGeneration)
                    this.ArgsToParamsOpt,
                    this.BinderOpt,
                    useSetterForDefaultArgumentGeneration,
+                   this.OriginalIndexersOpt,
```

Comment:
>this.OriginalIndexersOpt, [](start = 19, length = 25)

Could we replace current implementation of this method with an invocation of a standard Update method?